### PR TITLE
[API v4.2] Publish new methods, message and update types

### PR DESCRIPTION
**Pull request by kamikazechaser**
- [x] Documentation updated
- [x] Local test

### Description
Update the library with minor changes from the latest bot API update (v4.2 released on 26/07/2018).

Add new methods:
- **sendPoll**
- **stopPoll**

Add new message type:

- **poll**

Add new update type:

- **poll**

### References

- #718 
- [Update release notes from Telegram](https://core.telegram.org/bots/api#april-14-2019)

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2018 Yago
+Copyright (c) 2019 Yago
```
Comment (GochoMugo) [inline LICENSE.md]: Please leave out this change. This should be in a separate MR (or pushed to the main master branch instead).

File: README.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ Some things built using this library that might interest you:
 
 **The MIT License (MIT)**
 
-Copyright © 2018 Yago
+Copyright © 2019 Yago
```
Comment (GochoMugo) [inline README.md]: Same with this change.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-telegram-bot-api",
-  "version": "0.30.0",
+  "version": "0.31.0",
```
Comment (GochoMugo) [inline package.json]: Same with this change.

**Comment by kamikazechaser** [comment]
@GochoMugo  I will update the license/Readme in a separate MR. As for the version, bump it on npmjs and we can update the `package.json` simultaneously here.
